postcss.config.mjs

```diff
@@ -0,0 +1,6 @@
+import cssnano from 'cssnano';
+import sass from 'postcss-node-sass';
+
+export default {
+  plugins: [sass, cssnano],
```

Comment:
what do you think of adding autoprefixer here?

Reply:
See 25cc261cfa96aa27947c120545c70411be1ffc05.